From 5a6348c94e8f0971ea89291c3ccf1607a6c2e2c6 Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <Alejandro.Sierra@freescale.com>
Date: Wed, 4 Nov 2015 20:33:04 -0600
Subject: [PATCH 04/21] Add more configurations for the FreedomX R1.0 board

	- Support for spi-nor on FreedomX R1.0 board
	- Support for LPDDR2 interleave mode on FreedomX R1.0 board

Signed-off-by: Alejandro Sierra <Alejandro.Sierra@freescale.com>
---
 configs/mx6scm_1gb_freeX_spinor_defconfig              | 7 +++++++
 configs/mx6scm_1gb_interleaving_freeX_defconfig        | 5 +++++
 configs/mx6scm_1gb_interleaving_freeX_spinor_defconfig | 7 +++++++
 3 files changed, 19 insertions(+)
 create mode 100644 configs/mx6scm_1gb_freeX_spinor_defconfig
 create mode 100644 configs/mx6scm_1gb_interleaving_freeX_defconfig
 create mode 100644 configs/mx6scm_1gb_interleaving_freeX_spinor_defconfig

diff --git a/configs/mx6scm_1gb_freeX_spinor_defconfig b/configs/mx6scm_1gb_freeX_spinor_defconfig
new file mode 100644
index 0000000..37f36b8
--- /dev/null
+++ b/configs/mx6scm_1gb_freeX_spinor_defconfig
@@ -0,0 +1,7 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6scmfreeX/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6scm-1gb-freeX-rev1-ldo.dtb\""
+CONFIG_ARM=y
+CONFIG_TARGET_MX6SCMFREEX=y
+CONFIG_SYS_MALLOC_F=y
+CONFIG_SYS_MALLOC_F_LEN=0x400
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6scm_1gb_interleaving_freeX_defconfig b/configs/mx6scm_1gb_interleaving_freeX_defconfig
new file mode 100644
index 0000000..28faf3a
--- /dev/null
+++ b/configs/mx6scm_1gb_interleaving_freeX_defconfig
@@ -0,0 +1,5 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6scmfreeX/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6scm-1gb-freeX-rev1-ldo.dtb\",INTERLEAVING_MODE"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6SCMFREEX=y
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6scm_1gb_interleaving_freeX_spinor_defconfig b/configs/mx6scm_1gb_interleaving_freeX_spinor_defconfig
new file mode 100644
index 0000000..c13d8b1
--- /dev/null
+++ b/configs/mx6scm_1gb_interleaving_freeX_spinor_defconfig
@@ -0,0 +1,7 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6scmfreeX/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6scm-1gb-freeX-rev1-ldo.dtb\",INTERLEAVING_MODE"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6SCMFREEX=y
+CONFIG_SYS_MALLOC_F=y
+CONFIG_SYS_MALLOC_F_LEN=0x400
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
-- 
1.9.1

